x/ecocredit/client/basket/tx.go

```diff
@@ -36,33 +36,28 @@ func TxCreateBasketCmd() *cobra.Command {
 		Long: strings.TrimSpace(`Creates a bank denom that wraps credits
 
 Parameters:
-		name: the name used to create a bank denom for this basket token.
+
+- name:  the name used to create a bank denom for this basket token
```

Comment:
This format improves the generated documentation rather than everything being bundled into a paragraph.